genesys.py

```diff
@@ -213,6 +213,12 @@ def calculate_defense(character) :
 
 	return 0
 
+def purify_name(unclean_name):
+	keepcharacters = (' ','.','_')
+	purified_name = "".join(c for c in unclean_name if c.isalnum() or c in keepcharacters).rstrip()
+	return purified_name
+
+
```

Comment:
Handles the case where names have " or other non filename safe characters in them.